pact/coin-contract/coin.pact

```diff
@@ -72,6 +72,12 @@
     \the transaction, and the price is the spot price of gas at that time.    \
     \The gas buy will be executed prior to executing SENDER's code."
 
+    @model [ (property (> total 0.0))
+             (property (not (= sender ""))) ]
+
+    (enforce (not (= sender ""))
+      "sender name must be non-empty")
```

Comment:
Quick request - i learned this recently: Pact has `!=`. Could you replace `not (= ...)` where ever you see it?